nsls2tools/csx1/plans/nanop.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from cycler import cycler
+import bluesky.plans as bp
+
+### additional generic scans for nanopositioners.
+### ZP suffix in plan name is indicative of 4 motor parameters until the epics pseudo motor is made, then the generic non-ZP scan should be used.
+
+def spiral_continuous(detectors, x_motor, y_motor, x_start, y_start, npts, probe_size, overlap=0.8 , *, tilt=0.0, per_step=None, md=None):
+    '''Continuously increasing radius spiral scan, centered around (x_start, y_start)
+    which is generic regarding motors and detectors.
+
+    Parameters
+    ----------
+    x_motor : object
+        any 'setable' object (motor, etc.)
+    y_motor : object
+        any 'setable' object (motor, etc.)
+    x_start : float
+        x center
+    y_start : float
+        y center
+    npts : integer
+        number of points
+    probe_size : float
+        radius of probe in units of motors
+    overlap : float
+        fraction of probe overlap
+    ----------------------------------------------------------------
```

Comment:
Ok.  we can deleted the dashes and just make a note for the parameter.

Also, since these plans are technically CSX standard scan plans (and those that will go in this subdirectory in the future), how do we feel about adding the olog statement here so that the user does not need to decorate their overall plan? Everyone can comment: @stuwilkins @cmazzoli @wenhu @danielballan @tacaswell  @stuartcampbell  and other interested parties.